components/PrintableForm/PrintableForm.tsx

```diff
@@ -0,0 +1,59 @@
+import s from './PrintableForm.module.scss';
+import { Form } from 'data/flexibleForms/forms.types';
+
+interface Props {
+  form: Form;
+}
+
+const PrintableForm = ({ form }: Props): React.ReactElement => (
+  <div className={s.form}>
+    <h1>{form.name}</h1>
+    <p>* indicates a required question</p>
+    {form.steps.map((step, i) => (
+      <div className={s.step} key={step.name}>
+        <h2>
+          {i + 1}. {step.name}
+        </h2>
+        <p>{step.intro}</p>
+        {step.fields.map((field) => (
+          <>
+            <h3>
+              {field.question}
+              {field.required && '*'}
+            </h3>
+            <p>{field.hint}</p>
```

Comment:
Hint could be confusing, think an unrequird field that isn't inserted. The hint looks a bit like the answer on the form.

Reply:
i don't think i understand! i've tweaked the spacing so that there's less gap between a field's elements

Reply:
![Screenshot from 2021-07-09 15-15-04](https://user-images.githubusercontent.com/29123700/125091630-7d961180-e0c8-11eb-9e6c-f7a223e86508.png)

I think it is fine now with the margins, it's clear the hint isn't part of the actual answer